client/jetpack-connect/authorize.js

```diff
@@ -76,6 +76,7 @@ import {
 	isSiteBlacklistedError as isSiteBlacklistedSelector,
 } from 'state/jetpack-connect/selectors';
 import getPartnerSlugFromQuery from 'state/selectors/get-partner-slug-from-query';
+import getPartnerIdFromQuery from 'state/selectors/get-partner-id-from-query';
```

Comment:
Nit: do we want to keep these sorted alphabetically? Hi @sirreal 👋 😉

Reply:
alphabetizing
always
be!

Reply:
I went ahead and swapped the order of the selectors, but didn't alphabetize them along with all imports, primarily because I noticed that there were other selectors that were not alphabetized along with all of the imports.